### ISSUE TITLE
webdriverJs attach to existing firefox session GET /session did not match a known command

### ISSUE THREAD

**Issue by tfaron** [SeleniumHQ/selenium#6862]
I have a webDriverJS session started with an instance of chrome and firefox browers running
i'm running the latest webdriveJS / GeckoDriver

Running from Chrome is fine when using seleniumSessionId 

The return from GET sessions seems to work and if i call get session/cea42e5e9ec91ace194d54d4336810ab which is a Chrome session but when I pass a Firefox session i get the followin error:

curl GET http://16.100.206.160:4444/wd/hub/session/1d95ab9a-0a80-4492-8aba-16b9e27271bc --noproxy "*"

curl: (6) Could not resolve host: GET
{
  "state": "unknown command",
  "sessionId": null,
  "class": "org.openqa.selenium.remote.Response",
  "value": {
    "additionalInformation": "\nDriver info: driver.version: unknown",
    "localizedMessage": "GET \u002fsession\u002f1d95ab9a-0a80-4492-8aba-16b9e27271bc did not match a known command\nBuild info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'\nSystem info: host: 'DESKTOP-KP0NDTR', ip: '16.100.206.160', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'\nDriver info: driver.version: unknown",
    "supportUrl": null,
    "systemInformation": "System info: host: 'DESKTOP-KP0NDTR', ip: '16.100.206.160', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'",
    "cause": null,
    "stackTrace": [
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null,
      null
    ],
    "suppressed": [
    ],
    "message": "GET \u002fsession\u002f1d95ab9a-0a80-4492-8aba-16b9e27271bc did not match a known command\nBuild info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'\nSystem info: host: 'DESKTOP-KP0NDTR', ip: '16.100.206.160', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'\nDriver info: driver.version: unknown",
    "class": "org.openqa.selenium.UnsupportedCommandException",
    "buildInformation": null
  },
  "status": 9
}

I validated the session is active:

`tfaron@dvm:~$ curl GET http://16.100.206.160:4444/wd/hub/sessions --noproxy "*"
curl: (6) Could not resolve host: GET
{
  "status": 0,
  "value": [
    {
      "id": "cea42e5e9ec91ace194d54d4336810ab",
      "capabilities": {
        "acceptInsecureCerts": false,
        "acceptSslCerts": false,
        "applicationCacheEnabled": false,
        "browserConnectionEnabled": false,
        "browserName": "chrome",
        "chrome": {
          "chromedriverVersion": "2.45.615291 (ec3682e3c9061c10f26ea9e5cdcf3c53f3f74387)",
          "userDataDir": "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\scoped_dir6448_16862"
        },
        "cssSelectorsEnabled": true,
        "databaseEnabled": false,
        "goog:chromeOptions": {
          "debuggerAddress": "localhost:50579"
        },
        "handlesAlerts": true,
        "hasTouchScreen": false,
        "javascriptEnabled": true,
        "locationContextEnabled": true,
        "mobileEmulationEnabled": false,
        "nativeEvents": true,
        "networkConnectionEnabled": false,
        "pageLoadStrategy": "normal",
        "platform": "Windows NT",
        "proxy": {
        },
        "rotatable": false,
        "setWindowRect": true,
        "strictFileInteractability": false,
        "takesHeapSnapshot": true,
        "takesScreenshot": true,
        "timeouts": {
          "implicit": 0,
          "pageLoad": 300000,
          "script": 30000
        },
        "unexpectedAlertBehaviour": "ignore",
        "version": "71.0.3578.98",
        "webStorageEnabled": true
      }
    },
    {
      "id": "1d95ab9a-0a80-4492-8aba-16b9e27271bc",
      "capabilities": {
        "acceptInsecureCerts": false,
        "browserName": "firefox",
        "browserVersion": "60.0.2",
        "moz:accessibilityChecks": false,
        "moz:geckodriverVersion": "0.23.0",
        "moz:headless": false,
        "moz:processID": 6220,
        "moz:profile": "C:\\Users\\Administrator\\AppData\\Local\\Temp\\rust_mozprofile.Q5twQXwXICW7",
        "moz:useNonSpecCompliantPointerOrigin": false,
        "moz:webdriverClick": true,
        "pageLoadStrategy": "normal",
        "platformName": "windows_nt",
        "platformVersion": "10.0",
        "rotatable": false,
        "timeouts": {
          "implicit": 0,
          "pageLoad": 300000,
          "script": 30000
        }
      }
    }
  ]` 

`[12:00:50] I/launcher - Running 1 instances of WebDriver
[12:00:50] I/attachSession - Using the selenium server at http://127.0.0.1:4444/wd/hub
[12:00:50] I/attachSession - Using session id - de052a07-b0bb-467c-bcdd-2b44ba62e423
[12:00:50] E/launcher - GET /session/de052a07-b0bb-467c-bcdd-2b44ba62e423 did not match a known command
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-KP0NDTR', ip: '16.100.206.160', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: driver.version: unknown
[12:00:50] E/launcher - UnsupportedOperationError: GET /session/de052a07-b0bb-467c-bcdd-2b44ba62e423 did not match a known command
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'DESKTOP-KP0NDTR', ip: '16.100.206.160', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_181'
Driver info: driver.version: unknown
    at WebDriverError (C:\svt-web-client\projects\client-ng\node_modules\selenium-webdriver\lib\error.js:27:5)
    at UnsupportedOperationError (C:\svt-web-client\projects\client-ng\node_modules\selenium-webdriver\lib\error.js:343:5)
    at Object.checkLegacyResponse (C:\svt-web-client\projects\client-ng\node_modules\selenium-webdriver\lib\error.js:546:15)
    at parseHttpResponse (C:\svt-web-client\projects\client-ng\node_modules\selenium-webdriver\lib\http.js:509:13)
    at doSend.then.response (C:\svt-web-client\projects\client-ng\node_modules\selenium-webdriver\lib\http.js:441:30)
    at process._tickCallback (internal/process/next_tick.js:109:7)
From: Task: WebDriver.attachToSession()
    at Function.attachToSession (C:\svt-web-client\projects\client-ng\node_modules\selenium-webdriver\lib\webdriver.js:695:24)
    at AttachSession.getNewDriver (C:\svt-web-client\projects\client-ng\node_modules\protractor\built\driverProviders\attachSession.js:37:58)
    at Runner.createBrowser (C:\svt-web-client\projects\client-ng\node_modules\protractor\built\runner.js:195:43)
    at q.then.then (C:\svt-web-client\projects\client-ng\node_modules\protractor\built\runner.js:339:29)
    at _fulfilled (C:\svt-web-client\projects\client-ng\node_modules\protractor\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\svt-web-client\projects\client-ng\node_modules\protractor\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\svt-web-client\projects\client-ng\node_modules\protractor\node_modules\q\q.js:796:13)
    at C:\svt-web-client\projects\client-ng\node_modules\protractor\node_modules\q\q.js:556:49
    at runSingle (C:\svt-web-client\projects\client-ng\node_modules\protractor\node_modules\q\q.js:137:13)
    at flush (C:\svt-web-client\projects\client-ng\node_modules\protractor\node_modules\q\q.js:125:13)
[12:00:50] E/launcher - Process exited with error code 199
'

However if I run GET /session/firefoxId/url it returns me the URL from Firefox

So basically i can start a Firefox session via webDriverJs but cannot run protractor to connect to an existing session due to this GET failire

**Comment by diemol** [comment]
This is a question rather than an issue. Please send questions to the [selenium user group](https://groups.google.com/forum/#!forum/selenium-users), or join us in the [IRC/Slack channel](https://seleniumhq.herokuapp.com) where the community can help you as well.

**Comment by diemol** [comment]
In addition, you are getting errors while using curl, I'd advise to double check how to use the tool.

**Comment by tfaron** [comment]
I think it might be more of an issue, but I will post in the other forum. Everything works fine passing seleniumSessionId of a Chrome instance but for Firefox it fails. Even if i add 'SeleniumSessionId: <something> inside the conf.js file i use.
